docker-compose.arm.yml

```diff
@@ -15,8 +15,10 @@ services:
     env_file:
       - llm-server/.env
     ports:
-      - 8002:8002
-      - 5678:5678
+      - 8002
+      - 5678
+    deploy:
+      replicas: 3
     depends_on:
```

Comment:
We cannot hardcode the replicas here, since we don't have to run multiple instances on make install.